pay_android/android/src/main/kotlin/io/flutter/plugins/pay_android/GooglePayHandler.kt

```diff
@@ -212,10 +226,14 @@ class GooglePayHandler(private val activity: Activity) :
      * Data](https://developers.google.com/pay/api/android/reference/object.PaymentData)
      */
     private fun handlePaymentSuccess(paymentData: PaymentData?) {
+        requireNotNull(loadPaymentDataResult) {
```

Comment:
looks strange - in case of loadPaymentDataResult is null u would receive an exception. probably the same situation, as u had before with using !! operator

Reply:
if we leave safe-call operator without this logic, then the user will just be in one place